kludges.js

```diff
@@ -375,6 +373,11 @@ function restoreMethod(obj, name) {
 }
 
 var signals;
+var swipeTrackers = [
+    Main.overview?._swipeTracker,
+    Main.wm._workspaceAnimation?._swipeTracker,
+    Main.overview?._overview?._controls?._workspacesDisplay?._swipeTracker
+].filter(t => t !== undefined);
```

Comment:
Is this ok to be done on the top level or should it be done in enable/init? I'm unsure when the fields in Main.overview and Main.wm are set.

Reply:
Good question - it's done at the top level so it can be accessed by `tiling.js` (e.g. - when re-enabling the overrides on 'overview' hidden signal) - otherwise I would have defined in `init`.

Reply:
> I'm unsure when the fields in Main.overview and Main.wm are set.

Oh, yes, they are set (by order, so are set first, well before referenced here)  :smiley: